app/Services/DependencyService.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\Dependency\Dependency;
+use App\Models\User\User;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\Log;
+use LogicException;
+
+class DependencyService
```

Comment:
I know they are all void but could you specify return typed to that affect? Stops accidentally changing things.

Reply:
Done!